File: scripts/commit-restore.py
import subprocess
import os

os.chdir('/vercel/share/v0-project')

try:
    # Stage the file
    subprocess.run(['git', 'add', 'components/mgl/MGLAdminShell.tsx'], check=True)
    print("[v0] Staged MGLAdminShell.tsx")
    
    # Commit with message
    subprocess.run(['git', 'commit', '-m', 'restore: clean working state from origin/main'], check=True)
    print("[v0] Committed with message: restore: clean working state from origin/main")
    
    # Push to origin
    subprocess.run(['git', 'push', 'origin', 'HEAD'], check=True)
    print("[v0] Pushed to GitHub successfully")
    
except subprocess.CalledProcessError as e:
    print(f"[v0] Git error: {e}")
except Exception as e:
    print(f"[v0] Error: {e}")

File: scripts/commit-restore.js
import { execSync } from 'child_process';

try {
  console.log("[v0] Staging MGLAdminShell.tsx...");
  execSync('git add components/mgl/MGLAdminShell.tsx', { cwd: '/vercel/share/v0-project' });
  
  console.log("[v0] Committing with message...");
  execSync('git commit -m "restore: clean working state from origin/main"', { cwd: '/vercel/share/v0-project' });
  
  console.log("[v0] Pushing to GitHub...");
  execSync('git push', { cwd: '/vercel/share/v0-project' });
  
  console.log("[v0] Successfully committed and pushed!");
} catch (error) {
  console.error("[v0] Error during commit/push:", error.message);
  process.exit(1);
}
